src/main/groovy/com/github/benmanes/gradle/versions/reporter/PlainTextReporter.groovy

```diff
@@ -58,18 +57,19 @@ class PlainTextReporter extends AbstractReporter {
   }
 
   private def writeHeader(printStream) {
-    printStream.println """
-      |------------------------------------------------------------
```

Comment:
As mentioned in the description, the embedded linefeeds caused inconsistent behavior on Windows. Splitting these into multiple println statements, should make the output consistent independent of OS.